### ISSUE TITLE
Can't compile on Xcode 12 GM, Mac Catalyst

### ISSUE THREAD

**Issue by jurex** [xmartlabs/Eureka#2076]
The latest release (5.3.0) does not build on Xcode 12 GM, when using Mac Catalyst as target. Error message:

`Type 'UIDatePickerStyle' has no member 'inline'`

<img width="820" alt="Screenshot 2020-09-17 at 17 15 30" src="https://user-images.githubusercontent.com/925257/93490982-715a8100-f909-11ea-84bf-68253df42cac.png">

**Comment by mats-claassen** [comment]
Does it work for you with the function like this?:

```swift
func configurePickerStyle(_ cell: DatePickerCell, _ mode: UIDatePicker.Mode = .dateAndTime) {
        cell.datePicker.datePickerMode = mode
        // For Xcode 11.4 and above
        #if swift(>=5.2)
            if #available(iOS 14.0, *) {
                #if swift(>=5.3) && !(os(OSX) || (os(iOS) && targetEnvironment(macCatalyst)))
                    cell.datePicker.preferredDatePickerStyle = .inline
                #else
                    cell.datePicker.preferredDatePickerStyle = .wheels
                #endif
            }
            else if #available(iOS 13.4, *) {
                cell.datePicker.preferredDatePickerStyle = .wheels
            }
        #endif
    }
```

**Comment by jurex** [comment]
yes @mats-claassen, I'm able to compile the project again. Thanks!
